src/config.rs

```diff
@@ -305,6 +305,32 @@ impl Config {
         self.write()?;
         Ok(())
     }
+    pub fn keys(&self) -> Vec<&str> {
```

Comment:
How about using enum for keys to make sure the list of keys are exhaustive?
I found `strum` crate and it seems useful: https://stackoverflow.com/a/55056427

Reply:
Would you mean to return enum instead of vec? Is there a good way to convert struct to enum?

Reply:
Got it. let me replace the other match with enum.

Reply:
Ahh, if we make it as enum, we need to follow the naming rule.
```
Variant `android_branches` should have CamelCase name, e.g. `AndroidBranches` [incorrect-ident-case] 
```

Reply:
https://docs.rs/strum/latest/strum/additional_attributes/index.html

`#[strum(serialize_all = "kebab-case")]` will solve it

Reply:
ah, sorry, it was snake_case, not kebab-case

Reply:
Yeah, I found this too.

https://docs.rs/strum_macros/latest/strum_macros/derive.EnumString.html#example-howto-use-enumstring